apps/api/src/app/workflows-v2/usecases/get-default-values-from-schema/get-default-values-from-schema-usecase.ts

```diff
@@ -0,0 +1,52 @@
+import { JSONSchemaDto } from '@novu/shared';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface ExtractDefaultsCommand {
+  jsonSchemaDto: JSONSchemaDto;
+}
+
+export class ExtractDefaultsUseCase {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:**
* Please align the use-case name to the filename to follow existing conventions. This ensures that one can find the relevant use-case via both filename and symbol lookup in the IDE.‏